Client/Client.Qt/Widgets/ChatHistory.hpp

```diff
@@ -66,7 +67,9 @@ private Q_SLOTS:
 
 private:
     bool                                        _alreadyScrolling = false;
-    std::int32_t                                _left             = -1;
+    std::int32_t                                _left = -1;
     std::unique_ptr<ScrollArea>                 _scrollArea;
     std::vector<std::unique_ptr<MessageWidget>> _messageList;
+    int                                         _userId = 0;
```

Comment:
it is better to store ID in int64 variables.

Reply:
also don't forget about memory alignment: put IDs after bool

Reply:
also userId should be in message, not in chathistory, because chat can be channel with several people

Reply:
> also userId should be in message, not in chathistory, because chat could be channel with several people

This is temporary until we have authorization and ID for a user